dojo/tools/tfsec/parser.py

```diff
@@ -0,0 +1,73 @@
+import json
+import hashlib
+from dojo.models import Finding
+
+
+class TFSecParser(object):
+    """
+    A class that can be used to parse the tfsec JSON report file
+    """
+
+    # table to match tfsec severity to DefectDojo severity
+    SEVERITY = {
+        "ERROR": "High",
+        "WARNING": "Medium",
+        "INFO": "Info",
+    }
+
+    def get_scan_types(self):
+        return ["TFSec Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "TFSec Scan"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Import JSON output for TFSec scan report."
+
+    def get_findings(self, filename, test):
+        data = json.load(filename)
+        dupes = {}
+        if data.get('results'):
+            for item in data['results']:
```

Comment:
```suggestion
        if 'results' not in data:
            raise ValueError("Incorrect TFSec scan, missing attribute 'results'")
        for item in data.get('results', []):
```
This is the safest way to do it